### DISCUSSION TITLE
Update function receives no data if errors are present on the response and data contains information

### DISCUSSION THREAD

**Discussion by lobo-tuerto** [Akryum/vue-apollo#345] [Q&A]
When receiving a response that contains both `data` an `errors` keys, the `update (data)` function receives an empty object. Even if the network response contains data inside `data`.

When receiving this as a response:

```js
{
  data: {...}
}
```

`update (data)` is populated just fine.

When receiving this as a response:

```js
{
  data: {...},
  errors: [{...}, {...}]
}
```

`update (data)` receives an empty object, even though data actually contains information.

Is this expected?

**Comment by Akryum** [comment]
See: https://www.apollographql.com/docs/react/api/react-apollo.html#graphql-config-options-errorPolicy
